build.fsx

```diff
@@ -126,14 +98,12 @@ Target.create
     "Pack"
     (fun _ ->
         let properties =
-            [ ("Version", nugetVersion)
-              ("Authors", authors)
+            [ ("Authors", authors)
               ("PackageProjectUrl", gitUrl)
               ("PackageTags", tags)
               ("RepositoryType", "git")
```

Comment:
these source control properties can be removed - sourcelink/dotnet.reproducible builds infer them correctly for you.